includes/widgets/icon-list.php

```diff
@@ -487,6 +487,62 @@ protected function register_controls() {
 			]
 		);
 
+		$this->add_responsive_control(
+			'icon_self_vertical_align',
+			[
+				'label' => esc_html__( 'Vertical Alignment', 'elementor' ),
+				'type' => Controls_Manager::CHOOSE,
+				'options' => [
+					'top' => [
+						'title' => esc_html__( 'Start', 'elementor' ),
+						'icon' => 'eicon-v-align-top',
+					],
+					'middle' => [
+						'title' => esc_html__( 'Center', 'elementor' ),
+						'icon' => 'eicon-v-align-middle',
+					],
+					'bottom' => [
+						'title' => esc_html__( 'End', 'elementor' ),
+						'icon' => 'eicon-v-align-bottom',
+					],
+				],
+				'default' => '',
+				'selectors_dictionary' => [
+					'top' => 'flex-start',
+					'middle' => 'center',
+					'bottom' => 'flex-end',
+				],
+				'selectors' => [
+					'{{WRAPPER}}' => '--icon-self-vertical-align: {{VALUE}};',
+				],
+			]
+		);
+
+		$this->add_responsive_control(
+			'icon_vertical_offset',
+			[
+				'label' => esc_html__( 'Adjust Vertical Position', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'condition' => [
+					'icon_self_vertical_align' => [ 'top', 'bottom' ],
+				],
+				'size_units' => [ 'px' ],
+				'default' => [
+					'unit' => 'px',
+					'size' => 0,
+				],
+				'range' => [
+					'px' => [
+						'min' => -15,
+						'max' => 15,
+					],
+				],
+				'selectors' => [
```

Comment:
condition ?
If the user chose ‘center’ (or left the default ‘center’ option on) the slider should not be available

Reply:
This is implemented on line 526—528, however I will remove this as per the updated requirements.

Reply:
Done in 2eddc0bd0.

Reply:
@TzviRabinovitch @lukecarbis What about tests?